Bubble sort algorithm

bertil291utn opened the pull request:
### We have completed two bubble sort algorithm methods.
## First method 
> It's regular bubble sort method which sorts from the smallest to highest value

## Second method
> Doing the same as the previous method, but with a yield and block elements, where you can wite your array list inside this block.


:computer: :rocket: 
We will be glad if you give us some feedback

Bekhzod96 committed:
Second_Method added

bubble survey method

bertil291utn committed:
Update first method

- Change file name - Add title - Change left and side in the block

bertil291utn committed:
Add stickler config files

bertil291utn committed:
Update README file

pelzolga123 reviewed (approved):
Approved.

pelzolga123 commented:
## Hi Team! ![](https://user-images.githubusercontent.com/38801423/62908122-703d4480-bd44-11e9-9115-42025ab7f341.png) 
### You did amazing work, project works perfectly :+1: Approved!
Before you merge to master, please kindly do next: 
* Add instructions on how to run your project to README file.
* Remove debugging instructions, this project should only contain methods.

Then go ahead and merge to master.

_Goodluck in your next project_ *Happy coding*

bertil291utn committed:
Remving debugging instructions
- Removing them bubble-sort-algorithm file

bertil291utn committed:
Update README file

bertil291utn committed:
Rubocop file
- Enablebydefault option to true

bertil291utn committed:
Update pending cops
- Rubocop file with new pending cops

bertil291utn committed:
Fix linters errors
- Change name file to snake_case
- Update `if` for `nest unless` for fist method
- Change curly braces for `do` and `when`
- Update condition var from `> 0` to `.positive?` element
